### ISSUE TITLE
Missing object_usage_linter due to line break

### ISSUE THREAD

**Issue by bersbersbers** [jimhester/lintr#507]
```
foo <- function() {
    unknown_function(1)

    unknown_function(
        1
    )

    unknown_function(1)
}
```

I get this in VS Code, while in my opinion, the middle call should also be detected.

![image](https://user-images.githubusercontent.com/12128514/88161609-7b654980-cc10-11ea-9096-9fd829e8dde5.png)
![image](https://user-images.githubusercontent.com/12128514/88161642-84eeb180-cc10-11ea-9b4c-c76d888894b1.png)

**Comment by infotroph** [comment]
A variant of this that bit me today:  multiline conditionals.

```
foo <- function() {

    # warns
    if (unknown_function()) NULL 

    # does not warn
    if (unknown_function()) {
    	NULL
    }

}
```

**Comment by AshesITR** [comment]
Thanks for reporting.
The issue is in `parse_check_usage` when multiline warnings are issued:

``` r
foo <- function() {

    # warns
    if (unknown_function()) NULL 

    # does not warn
    if (unknown_function()) {
        NULL
    }

}

codetools::checkUsage(foo)
#> <anonymous>: no visible global function definition for 'unknown_function' (<text>:4)
#> <anonymous>: no visible global function definition for 'unknown_function' (<text>:7-9)
lintr:::parse_check_usage(foo)
#>                                                        message             name
#> 1 no visible global function definition for 'unknown_function' unknown_function
#> 2                                                         <NA>             <NA>
#>     path line_number
#> 1 <text>           4
#> 2   <NA>        <NA>
```

<sup>Created on 2021-02-15 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>
